test/junit/scala/collection/parallel/TaskTest.scala

```diff
@@ -27,4 +27,13 @@ class TaskTest {
 
     for (x <- one ; y <- two) assert(Thread.currentThread.getName == "two")
   }
+
+  @Test
+  def `t152 pass on task support`(): Unit = {
```

Comment:
The `152` doesn't refer to scala/bugs but to scala/scala-parallel-collections. Let's put that in the description for future archaeologists.

Reply:
👍  thanks